modules/cart/commerce_cart.module

```diff
@@ -4,3 +4,30 @@
  * @file
  * Implements the shopping cart system and add to cart features.
  */
+use Drupal\Core\Entity\Entity\EntityViewMode;
+
+/**
+ * Implements hook_rebuild().
```

Comment:
Below the Implements line let's describe what this function does. "Creates a cart view mode for each line item source entity type."